common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -222,10 +230,20 @@ private void maybeStartReporting() {
         reportingThread.setDaemon(true);
         reportingThread.start();
 
-        metricsThread = Metrics.getInstance(auth.getAccessToken(),
-                LightStepConstants.Metrics.DEFAULT_INTERVAL_SECS,
-                LightStepConstants.Metrics.DEFAULT_HOST + LightStepConstants.Metrics.PATH,
-                LightStepConstants.Metrics.DEFAULT_PORT);
+        final int samplePeriodSeconds = LightStepConstants.Metrics.DEFAULT_INTERVAL_SECS;
+        final String componentName = auth.getAccessToken();
+        final String servicePath = LightStepConstants.Metrics.DEFAULT_HOST + LightStepConstants.Metrics.PATH;
+        final int servicePort = LightStepConstants.Metrics.DEFAULT_PORT;
+
+        final Sender<?,?> sender;
+        if (metricsCollectorClient == Options.CollectorClient.HTTP)
+          sender = new OkHttpSender(samplePeriodSeconds * 1000, componentName, servicePath, servicePort);
```

Comment:
sample periods is already in seconds, so I don't think we need to multiply it by 1000